app/src/main/res/values/colors.xml

```diff
@@ -2,7 +2,29 @@
 <resources>
     <color name="colorPrimary">#F0163A</color>
     <color name="colorPrimaryDark">#CC1231</color>
-    <color name="colorAccent">#FFD90f</color>
+    <color name="colorAccent">#FFD90F</color>
     <color name="grayBackground">#9e9e9e</color>
 
+    <color name="screen1">#E3F2FD</color>
```

Comment:
All of these new colors should probably be changed, to a black/white or very light gray, with the dots being colorAccent436 and colorAccentLight436